doc/Extensions/Services.md

```diff
@@ -4,39 +4,41 @@ path: blob/master/doc/
 
 # Setting up services
 
-Services within LibreNMS provides the ability to use Nagios plugins to
-perform additional monitoring outside of SNMP.
+Services within LibreNMS provides the ability to leverage Nagios plugins to
+perform additional monitoring outside of SNMP. Services can also be used
+in conjuction with your SNMP monitoring for larger monitoring functionality.
 
-**These services are tied into an existing device so you need at least
-one device to be able to add it to LibreNMS - localhost is a good
-one. This is needed in order for alerting to work properly.**
+**Services must be tied to a device to function properly. A good generic
+option is to use `localhost`, but it is suggest to attach the check to 
+the device you are monitoring.**
 
-## Pre installed plugins
+## Nagios plugins source
 
-Note: Plugins will only load if they are prefixed with "check_" and
-they have that prefix stripped out when displaying in the "Add Service"
-GUI "Type" dropdown list.
+Plugins come from two main sources:
 
-Plugins come from two main places
-[pkg-nagios-plugins-contrib](https://github.com/bzed/pkg-nagios-plugins-contrib)
-and [monitoring-plugins](https://www.monitoring-plugins.org). This is
-where you can find the documentation for most, if not all of the plugins.
+* [monitoring-plugins](https://www.monitoring-plugins.org)
+* [pkg-nagios-plugins-contrib](https://github.com/bzed/pkg-nagios-plugins-contrib)
 
-The plugins are bundled with the pre build VM images via the package
-`monitoring-plugins` in Ubuntu and `nagios-plugins-all` in CentOS.
+Note: Plugins will only load if they are prefixed with `check_`.
+The `check_` prefix is stripped out when displaying in the "Add Service"
+GUI "Type" dropdown list.
 
 ## Setup
 
-Service checks is now distributed aware. If you run a distributed
-setup then you can now run `services-wrapper.py` in cron instead of
-`check-services.php` across all polling nodes.
+Service checks are now distributable if you run a distributed
+setup. To leverage this, use `services-wrapper.py` in cron instead of
```

Comment:
The dispatch service also works with services, not just the wrapper.

Reply:
Please let me know if the new bit about the `dispatcher` service is functional for this.

Reply:
yes :)